docs/getting_started/tutorial.md

```diff
@@ -921,8 +921,8 @@ class BlogTagIndexPage(Page):
 Note that this Page-based model defines no fields of its own.
 Even without fields, subclassing `Page` makes it a part of the
 Wagtail ecosystem, so that you can give it a title and URL in the
-admin, and so that you can manipulate its contents by returning
-a QuerySet from its `get_context()` method.
+admin and manipulate its contents by overridding its `get_context()`
```

Comment:
"Overriding" is misspelled here:
```suggestion
admin and manipulate its contents by overriding its `get_context()`
```

Reply:
FWIW, I'm in two minds about removing the mention of the queryset. "returning a QuerySet" is certainly inaccurate, but it's an attempt to explain the intent behind overriding `get_context` rather than just saying "we're overriding it" without elaboration. Maybe something like "manipulate its contents by adding a QuerySet as part of the items returned from its `get_context()` method"...?

I feel like the whole sentence is a little bit muddled though. It starts by introducing the idea that a page with no fields can still be meaningful, and then goes on to talk about `get_context` which is not really connected to that first idea. "Manipulating its contents" also seems like a weird way to refer to outputting some data onto the template. But at that point we're getting into much deeper rewrites that what's being proposed here, so maybe that's better left for another time...

Reply:
Thank you @gasman for the review. I reworded the note to mention the goal of `get_context()` override (adding a QuerySet to the context), and also took the liberty to make that last clause its own sentence, as per your comment.